mountpoint-s3/src/cli.rs

```diff
@@ -293,6 +293,16 @@ pub struct CliArgs {
         value_name = "ALGORITHM",
     )]
     pub upload_checksums: Option<UploadChecksums>,
+
+    #[cfg(feature = "multiple-nw-iface")]
+    #[clap(
+        long,
+        help = "One or more comma-delimited network interfaces for Mountpoint to use when accessing S3. Requires Linux 5.7+ or running as root.",
+        help_heading = CLIENT_OPTIONS_HEADER,
+        value_delimiter = ',',
+        value_name = "INTERFACE_NAMES",
+    )]
+    pub network_interfaces: Option<Vec<String>>,
```

Comment:
I don't love comma-delimited arguments. I kind of think we should do this Docker-style, with multiple uses of the argument (and a shorter name):
```
--bind eth0 --bind eth1 --bind eth2
```